### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by dustMason** [bryanbraun/after-dark-css#3]
These are so cool! What's the license? I'd like to use them on my site

**Comment by bchar** [comment]
Im sorry as of today, Im not sure where I got the given a few to go back in time to research it. Sorry

**Comment by bryanbraun** [comment]
Honestly, the license/reuse situation is a bit murky.

I'm happy to license the code (HTML & CSS) permissively with MIT or another license. My code is all original and the [ChicagoFLF font I used](http://christtrekker.users.sourceforge.net/fnt/chicago.shtml) already has an open license (the SIL Open Font License).

Many of the images, while not straight copies of the original After Dark images, are very close to the originals, and would probably retain their original copyright status.

There are a number of reasons this might be ok. The project is a limited set of screensaver approximations, with very limited functionality. It's also a non-commercial, personal project, thus, this could be considered fair-use of the images. Other similar, more visible projects, like [Simpsons in CSS](http://pattle.github.io/simpsons-in-css/), are in similar legal territory without running into issues.

You could also consider it an archival project. Archive.org collects lots of old software, and avoids copyright issues by relying on a DMCA exception that applies to "Computer programs and video games distributed in formats that have become obsolete and which require the original media or hardware as a condition of access." ([more details here](http://archive.org/about/dmca.php)).

That being said, the flying toasters images specifically [have been the subject of lawsuits in the past](https://en.wikipedia.org/wiki/After_Dark_%28software%29#Flying_Toasters). Of course, that was back in the day when the screensavers were sold, and copyright infringement had a real financial impact). If I were asked to take down the project, I would immediately comply.

In short, I'm comfortable posting the project under the above circumstances, and I'm comfortable with others embedding them, repurposing them, and making derivative works, but they ought to recognize the possibility that the images could create some degree of legal risk.

**Comment by bryanbraun** [comment]
I documented the license details in the README as part of https://github.com/bryanbraun/after-dark-css/commit/405af9d6abb85a50dcb82df3cbe342f2cefd36f7. Yay for finally closing this 7-year old issue! 😄
